osu.Game/Overlays/SkinEditor/SkinEditor.cs

```diff
@@ -465,18 +482,16 @@ private void populateSettings()
                 settingsSidebar.Add(new SkinSettingsToolbox(component));
         }
 
-        private IEnumerable<SkinComponentsContainer> availableTargets => targetScreen.ChildrenOfType<SkinComponentsContainer>();
+        private IEnumerable<ISerialisableDrawableContainer> availableTargets => targetScreen.ChildrenOfType<ISerialisableDrawableContainer>()
```

Comment:
The dropdown seemed to not always update correctly (I guess it's going to be hard to get this correct as it wasn't made to be updated as components load in). Also I'm not sure about having one entry per component so we'd have to find a way to make that work if we go with this structure.

https://github.com/ppy/osu/assets/191335/637f078a-1f9c-4163-b047-a8ea483ed896